ui/README.md

```diff
@@ -4,34 +4,52 @@ This directory contains the web UI for ORT Server.
 
 ## Architecture
 
-The UI is a [React](https://react.dev/) application and uses [Vite](https://vitejs.dev/) as the
-build tool and [pnpm](https://pnpm.io/) as the package manager.
+The UI is a [React](https://react.dev/) application and uses [Vite](https://vitejs.dev/) as the build tool and [pnpm](https://pnpm.io/) as the package manager.
 
-## Development
+## Prerequisites
 
 The UI expects ORT Server to be running locally.
```

Comment:
This sentence also needs clarification, it sounds like the UI could not connect to a remote instance. Maybe just by prefixing with "By default, the UI expects...".